core/os/os.cpp

```diff
@@ -357,13 +361,7 @@ void OS::set_has_server_feature_callback(HasServerFeatureCallback p_callback) {
 
 bool OS::has_feature(const String &p_feature) {
 	// Feature tags are always lowercase for consistency.
-	if (p_feature == get_name().to_lower()) {
-		return true;
-	}
-
-	// Catch-all `linuxbsd` feature tag that matches on both Linux and BSD.
-	// This is the one exposed in the project settings dialog.
-	if (p_feature == "linuxbsd" && (get_name() == "Linux" || get_name() == "FreeBSD" || get_name() == "NetBSD" || get_name() == "OpenBSD" || get_name() == "BSD")) {
+	if (p_feature == get_identifier()) {
```

Comment:
This change breaks .gdextension files that have libraries like:

```
linux.debug.x86_64 = "res://bin/libexample.so"
```

... you now have to change that to:

```
linuxbsd.debug.x86_64 = "res://bin/libexample.so"
```

This would also affect project settings that used the ".linux" prefix - those would also need to update to ".linuxbsd".

Is this breakage that we want? Or, was this unintentional?

Reply:
I'm gonna assume this was unintentional :-)

Here's a PR to fix it: https://github.com/godotengine/godot/pull/76974